plugin/trino-loki/src/test/java/io/trino/plugin/loki/TestLokiIntegration.java

```diff
@@ -174,22 +177,30 @@ public void testSelectTimestamp()
                         LIMIT 1
                         """, timestampFormatter.format(start), timestampFormatter.format(end)),
                 String.format("VALUES ('%s', 'line 1')", isoTimestampFormatter.format(firstLineTimestamp)));
+    }
+
+    @Test
+    public void testTimestampMetricsQuery()
+            throws Exception
+    {
+        LocalDate baseLineDate = LocalDate.now();
+        Instant start = Instant.ofEpochMilli(baseLineDate.toEpochDay() * MILLISECONDS_PER_DAY);
```

Comment:
Shouldn't this be the same as `Instant.now().truncatedTo(ChronoUnit.DAYS)` above?